src/ui/public/error_url_overflow/error_url_overflow.html

```diff
@@ -1,32 +1,46 @@
 <div class="app-container kbnError--url-overflow-app euiText">
-  <h3><icon aria-hidden="true" type="'alert'" color="'danger'"></icon> Woah there!</h3>
-  <p>
-    That's a big URL you have there. I have some unfortunate news: Your browser doesn't play nice
+  <h3><icon aria-hidden="true" type="'alert'" color="'danger'"></icon>
+    <span i18n-id="common.ui.errorUrlOverflow.errorTitle"
+    i18n-default-message="Woah there!"></span>
+  </h3>
+
+  <p i18n-id="common.ui.errorUrlOverflow.errorDescription"
+    i18n-default-message="That's a big URL you have there. I have some unfortunate news: Your browser doesn't play nice
     with Kibana's bacon-double-cheese-burger-with-extra-fries sized URL. To keep you from running
-    into problems Kibana limits URLs in your browser to {{ controller.limit }} characters for your
-    safety.
-  </p>
+    into problems Kibana limits URLs in your browser to {urlCharacterLimit} characters for your
+    safety."
+    i18n-values="{ urlCharacterLimit: controller.limit }"
+  ></p>
+
+  <h3 i18n-id="common.ui.errorUrlOverflow.fixErrorTitle"
+    i18n-default-message="Ok, how do I fix this?"
+  ></h3>
+  <p i18n-id="common.ui.errorUrlOverflow.fixErrorListDescription"
+    i18n-default-message="This usually only happens with big, complex dashboards, so you have some options:"
+  ></p>
 
-  <h3>Ok, how do I fix this?</h3>
-  <p>This usually only happens with big, complex dashboards, so you have some options:</p>
   <ol>
-    <li>
-      Enable the <code>state:storeInSessionStorage</code> option in the
-      <a href="#/management/kibana/settings">advanced settings</a>. This will prevent the URLs from
-      getting long, but makes them a bit less portable.
-    </li>
-    <li>
-      Remove some stuff from your dashboard. This will reduce the length of the URL and keep IE in a
-      good place.
-    </li>
-    <li>Don't use IE. Every other supported browser we know of doesn't have this limit.</li>
+    <li i18n-id="common.ui.errorUrlOverflow.fixErrorListItemOne"
```

Comment:
please do not use ItemOne, ItemTwo in ids. Add more descriptive id which describes message